sass/components/inline-grid.scss

```diff
@@ -178,18 +178,18 @@
 }
 
 // Column Sizes
-@for $i from 0 through $grid-column-size {
+@for $i from 0 through $grid-column-max {
```

Comment:
@patrickdmccarthy i'm not sure this is doing what we think it's doing.

I think this is making a grid that as as many columns as the max. Where as CSS grid has true column span, I think this is just setting a percentage of the max. 

I think in order to get multiple grids we need multiple loops :/ 

Thoughts?

Reply:
Sure we can do that – I did this to replicate the existing behavior since we currently only support 1 inline grid by default. 

If we want to continue adding features to the inline-grid, i can add the loop, or we can say that if there are multiple grids, default to css grid.  I'm fine either way, just let me know what you'd like to do here.